exist-core/src/test/resources-filtered/org/exist/storage/statistics/conf.xml

```diff
@@ -712,7 +712,7 @@
 
         <!-- html-to-xml class="org.ccil.cowan.tagsoup.Parser"/ -->
 
-        <html-to-xml class="org.cyberneko.html.parsers.SAXParser">
+        <html-to-xml class="org.codelibs.nekohtml.parsers.SAXParser">
             <properties>
                 <property name="http://cyberneko.org/html/properties/names/elems" value="match"/>
```

Comment:
@dizzzz As the package name has changed, I am wondering if these properites have also been renamed?

Reply:
Thanks @dizzzz :-)